Hosts/Console/TddStud10App/TddStud10App.csproj

```diff
@@ -180,6 +189,7 @@
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
+    <None Include="TddStud10AppCodeAnalysisRules.ruleset" />

```

Comment:
There is nothing preventing us from doing this. I just didn't want to make that decision, but will put it in the solution files?